traveler-swift-core/TravelerKit/Models/ParkingQuery.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  ParkingQuery.swift
+//  TravelerKit
+//
+//  Created by Omar Padierna on 2019-10-05.
+//  Copyright © 2019 Ata Namvari. All rights reserved.
+//
+
+import Foundation
+
+/// A query to fetch Parking items
+public struct ParkingQuery {
+    /// Pagination offset
+    public let offset: Int
+    /// Pagination limit
+    public let limit: Int
+    /// IATA code as string for aiport oriented queries
+    public let airport: String?
```

Comment:
we have an Airport model....call this `airportIATA` or something so its not confusing with an actual Airport type